packages/mongo-schema/types.js

```
@@ -0,0 +1,102 @@
+  bsonType: 'bool',
+});
+
+TYPE_MAP.set(Date, {
+  name: 'date',
+  check: (v) => v instanceof Date && !isNaN(v.getTime()),
+  bsonType: 'date',
+});
+
+TYPE_MAP.set(Object, {
+  name: 'object',
+  check: (v) => v !== null && typeof v === 'object' && !Array.isArray(v) && !(v instanceof Date),
```

ESLint

[error] Replace `(v)·=>·v·!==·null·&&·typeof·v·===·'object'·&&·!Array.isArray(v)·&&` with `v·=>⏎····v·!==·null·&&⏎····typeof·v·===·'object'·&&⏎····!Array.isArray(v)·&&⏎···`

(prettier/prettier)